client/src/components/Screens/Login/Login.jsx

```diff
@@ -23,16 +23,22 @@ const Login = () => {
         email,
         password,
       });
+      console.log(response);
```

Comment:
Technically should remove this from your final code, should be fine now

Reply:
yeah, understood